test_site/config.toml

```diff
@@ -22,6 +22,8 @@ taxonomies = "on"
 anchors = "on"
 
 [link_checker]
+internal_level = "error"
+external_level = "error"
```

Comment:
That's the default, so let's not add those in the test site

Reply:
Done :+1: